backend/src/main/java/gov/cdc/usds/simplereport/api/model/ApiTestOrder.java

```diff
@@ -16,13 +16,6 @@ public Date getDateAdded() {
     return wrapped.getCreatedAt();
   }
 
-  public String getResult() {
```

Comment:
I replaced it with a schema mapping to get the result, that way we get null instead of an empty string, or string value from a `TestResult` enum

https://github.com/CDCgov/prime-simplereport/blob/dc1a9e2f3a99b6cfcc811ee57e8eb2c635baf6b7/backend/src/main/java/gov/cdc/usds/simplereport/api/queue/ApiTestOrderDataResolver.java#L109-L117

https://github.com/CDCgov/prime-simplereport/blob/dc1a9e2f3a99b6cfcc811ee57e8eb2c635baf6b7/backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/TestResult.java#L3-L7